gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/writer/MetadataWriterKeys.java

```diff
@@ -15,33 +15,37 @@
  * limitations under the License.
  */
 
-package org.apache.gobblin.iceberg.writer;
+package org.apache.gobblin.hive.writer;
 
-public class IcebergMCEMetadataKeys {
+public class MetadataWriterKeys {
   public static final String METRICS_NAMESPACE_ICEBERG_WRITER = "IcebergWriter";
   public static final String ICEBERG_COMMIT_EVENT_NAME = "IcebergMetadataCommitEvent";
+  public static final String HIVE_COMMIT_EVENT_NAME = "HiveMetadataCommitEvent";
   public static final String METADATA_WRITER_FAILURE_EVENT = "MetadataWriterFailureEvent";
   public static final String LAG_KEY_NAME = "endToEndLag";
   public static final String SNAPSHOT_KEY_NAME = "currentSnapshotId";
   public static final String MANIFEST_LOCATION = "currentManifestLocation";
   public static final String SNAPSHOT_INFORMATION_KEY_NAME = "currentSnapshotDetailedInformation";
-  public static final String TABLE_KEY_NAME = "icebergTableName";
-  public static final String DATABASE_KEY_NAME = "icebergDatabaseName";
+  public static final String ICEBERG_TABLE_KEY_NAME = "icebergTableName";
+  public static final String ICEBERG_DATABASE_KEY_NAME = "icebergDatabaseName";
   public static final String GMCE_TOPIC_NAME = "gmceTopicName";
   public static final String GMCE_TOPIC_PARTITION = "gmceTopicPartition";
   public static final String GMCE_HIGH_WATERMARK = "gmceHighWatermark";
   public static final String GMCE_LOW_WATERMARK = "gmceLowWatermark";
   public static final String DATASET_HDFS_PATH = "datasetHdfsPath";
-  public static final String FAILURE_EVENT_DB_NAME = "databaseName";
-  public static final String FAILURE_EVENT_TABLE_NAME = "tableName";
+  public static final String PARTITION_HDFS_PATH = "partitionHdfsPath";
+  public static final String DATABASE_NAME_KEY = "databaseName";
```

Comment:
Change this to hiveDatabaseName? Or can we migrate ICEBERG_TABLE_KEY_NAME to use this one as well?

Reply:
I added hive database name as a new key since that key is also used in sending failure events which is not specific to hive or iceberg, and I don't think I want to migrate the iceberg key names for backwards compatibility.,